packages/preview/ascii-ipa-1.0.0/typst.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "ascii-ipa"
+version = "1.0.0"
+entrypoint = "lib.typ"
+authors = ["imatpot"]
+license = "MIT"
+description = "Converter for ASCII representations for the International Phonetic Alphabet (IPA)"
```

Comment:
one more thing: should it be "for the International Phonetic Alphabet" or maybe rather "of the International Phonetic Alphabet" here? I'm not sure.

Reply:
I guess "of" makes slightly more sense